scripts/eval/eval.py

```diff
@@ -131,23 +135,21 @@ def main(cfg: DictConfig):
     models_df = None
     composite_scores = None
     for model_cfg in cfg.models:
-        (in_memory_logger, logger_keys, model_gauntlet_callback, model_gauntlet,
+        (logger, logger_keys, model_gauntlet_callback, model_gauntlet,
          model_gauntlet_df) = evaluate_model(model_cfg, cfg, cfg.run_name,
                                              model_gauntlet_df)
 
         if model_gauntlet_callback is not None:
-            # TODO(bmosaicml) This needs to be refactored to fix the typing issue
-            composite_scores = model_gauntlet_callback.eval_end(
-                None, in_memory_logger)  # type: ignore
+            composite_scores = model_gauntlet_callback.eval_after_all(
+                None, logger)
 
         benchmark_to_taxonomy = {}
         if model_gauntlet is not None:
             for t in model_gauntlet.categories:
                 for b in t.benchmarks:
                     benchmark_to_taxonomy[b.name] = t.name
 
-        model_results = calculate_markdown_results(logger_keys,
-                                                   in_memory_logger.data,
+        model_results = calculate_markdown_results(logger_keys, logger,
```

Comment:
why are we using markdown results? This seems like an odd choice for outputting structured data

Reply:
I'm open to other suggestions. What did you have in mind?